From cb8c7690048946dd298371876093997f07785269 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 7 Feb 2017 14:56:10 +0100
Subject: [PATCH 11/11] HMP: Fix documentation of __com.redhat.drive_add

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1486479370-24026-3-git-send-email-armbru@redhat.com>
Patchwork-id: 73592
O-Subject: [RHEL-7.4 qemu-kvm PATCH 2/2] HMP: Fix documentation of __com.redhat.drive_add
Bugzilla: 1419898
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>

It's similar to -drive if=none, not -device if=none.  Screwed up in
RHEL-6.0 commit 545d0d8, forward-ported to RHEL-7.0 in commit c18bb50.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hmp-commands.hx | 4 ++--
 qmp-commands.hx | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hmp-commands.hx b/hmp-commands.hx
index a8ba626..5356c4c 100644
--- a/hmp-commands.hx
+++ b/hmp-commands.hx
@@ -1105,7 +1105,7 @@ ETEXI
         .name       = RFQDN_REDHAT "drive_add",
         .args_type  = "simple-drive:O",
         .params     = "id=name,[file=file][,format=f][,media=d]...",
-        .help       = "Create a drive similar to -device if=none.",
+        .help       = "Create a drive similar to -drive if=none.",
 	.user_print = monitor_user_noop,
         .mhandler.cmd_new = simple_drive_add,
     },
@@ -1113,7 +1113,7 @@ ETEXI
 STEXI
 @item __com.redhat_drive_add
 @findex __com.redhat_drive_add
-Create a drive similar to -device if=none.
+Create a drive similar to -drive if=none.
 ETEXI
 
 #if defined(TARGET_I386) && 0 /* Disabled for Red Hat Enterprise Linux */
diff --git a/qmp-commands.hx b/qmp-commands.hx
index 9522c44..4a89c24 100644
--- a/qmp-commands.hx
+++ b/qmp-commands.hx
@@ -110,7 +110,7 @@ EQMP
         .name       = RFQDN_REDHAT "drive_add",
         .args_type  = "simple-drive:O",
         .params     = "id=name,[file=file][,format=f][,media=d]...",
-        .help       = "Create a drive similar to -device if=none.",
+        .help       = "Create a drive similar to -drive if=none.",
 	.user_print = monitor_user_noop,
         .mhandler.cmd_new = simple_drive_add,
     },
@@ -119,7 +119,7 @@ SQMP
 __com.redhat_drive_add
 ----------------------
 
-Create a drive similar to -device if=none.
+Create a drive similar to -drive if=none.
 
 Arguments:
 
-- 
1.8.3.1

